params/config.go

```diff
@@ -72,6 +73,7 @@ var (
 		IstanbulCompatibleBlock:  big.NewInt(75373312),
 		LondonCompatibleBlock:    big.NewInt(80295291),
 		EthTxTypeCompatibleBlock: big.NewInt(86513895),
+		KIP71CompatibleBlock:     big.NewInt(999999999), // TODO-Klaytn-KIP71: Set HF blocknumber correctly
```

Comment:
@aidan-kwon What about this? In previous hard-forks, the block number was `nil` before setting correct hardfork block number.
```suggestion
		KIP71CompatibleBlock:     nil, // TODO-Klaytn-KIP71: Set HF blocknumber correctly
```